boards/waveshare/esp32s3_poe_eth_8di_8ro/board.cmake

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2024 Joel Guittet
```

Comment:
If this was copied from the Espressif boards then you should preserve the copyright message.

Reply:
You're right, I didn't notice that.

Reply:
So we keep it like this?